CSS/styles.css

```diff
@@ -0,0 +1,215 @@
+/* --------------------------- Base styles --------------------------- */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Arial', sans-serif;
+    -webkit-font-smoothing: antialiased; /* do I need this? */
+    background: #FFF;
+    color:#222;
+}
+
+ul {
+    list-style: none;
+}
+
+/* 
+h1, h2, h3, h3, h4 {
+
+}
+*/
+
+a {
+    color: #222;
+    text-decoration: none;
+}
+
+
+p {
+    margin: 0.5rem 0;
+}
+
+img {
+    width: 100%;
+}
+
+.content-wrap {
+    width: 90%;
+    margin: 0 auto;
+}
+
+/* Buttons */
+.btn {
+    display: inline-block;
+    cursor: pointer;
+    border: 1px solid #DDD;
+    border-radius: 4px;
+    background-color: #FFF;
+}
+
+.btn:hover {
+
+}
+
+.btn-rounded {
+	border-radius: 4px;
+}
+
+/* --------------------------- Header --------------------------- */
+header {
+    width: 100%;
+    height: 178px;
+    position: fixed;
```

Comment:
This is good for the header but there are a few comments on the list of cards and the footer.